source/jobs/stopping-jobs.txt

```diff
@@ -43,22 +45,39 @@ To stop your sync job, follow the instructions below based on the current stage
            name: Stop a Sync Job in Snapshot stage
            content: |
 
-              #. Open the :guilabel:`Data Migration` tab.
-              #. Click on the sync job in the :guilabel:`History` pane.
-              #. Click the red :guilabel:`Terminate` button in the upper-right corner of the screen.
-              #. Click the red :guilabel:`Terminate` button on the pop-up modal.
-                 Once a sync job cancels, an :guilabel:`x` icon displays 
-                 in the :guilabel:`History` pane, and the status of the sync
-                 job updates to :guilabel:`Snapshot cancelled`.
+              .. procedure::
+                :style:  connected
+
+                .. step:: Open the :guilabel:`Data Migration` tab
+
+                .. step:: Click on the sync job in the :guilabel:`History` pane
+
+                .. step:: Stop the sync job 
+
+                   a. Click the red :guilabel:`Terminate` button in the upper-right corner of the screen.
+                   #. Click the red :guilabel:`Terminate` button on the pop-up modal.
+                      Once a sync job cancels, an :guilabel:`x` icon displays 
+                      in the :guilabel:`History` pane, and the status of the sync
+                      job updates to :guilabel:`Snapshot cancelled`.
 
          - id: stop-cdc
            name: Stop a Sync Job in CDC stage
            content: |
-              #. Open the :guilabel:`Data Migration` tab.
-              #. Click on the sync job in the :guilabel:`History` pane.
-              #. Click the :guilabel:`Complete CDC`  button in the upper-right corner of the screen.
-              #. Click the green :guilabel:`Complete CDC` button on the pop-up modal.
-                 Once the sync job completes, a checkmark icon displays in the :guilabel:`History` pane, all stages display a status of :guilabel:`COMPLETED`, and the status of the sync job updates to :guilabel:`CDC completed`.
+              .. procedure::
+                :style: connected
+
+                .. step:: Open the :guilabel:`Data Migration` tab
+
+                .. step:: Click on the sync job in the :guilabel:`History` pane
+
+                .. step:: Complete the sync job 
+
+                   a. Click the :guilabel:`Complete CDC` button in the upper-right corner of the screen.
+                   #. Click the green :guilabel:`Complete CDC` button on the pop-up modal.
+                      After completing, the sync job shows a checkmark 
+                      in the :guilabel:`History` pane, marks all stages
+                      as :guilabel:`COMPLETED`, and updates its status 
```

Comment:
[nit]
[Per the style guide:](https://www.mongodb.com/docs/meta/style-guide/style/text-formatting) If you are representing all-cap text from the user interface, write it as initial cap instead.

```suggestion
                      as :guilabel:`Completed`, and updates its status 
```